pkg/common/args.go

```diff
@@ -144,22 +46,23 @@ func PrintInfo(clientSet *kubernetes.Clientset, config *rest.Config) {
 
 // ValidateArgs validates the arguments passed to the program
 // and creates the output directory if it doesn't exist
-func ValidateArgs(clientSet *kubernetes.Clientset, config *rest.Config, cfg *ArgConfig) {
-	if cfg.ConformanceTests {
-		cfg.Focus = "\\[Conformance\\]"
+func ValidateArgs(clientSet *kubernetes.Clientset, config *rest.Config) {
+	if viper.Get("focus") == "" {
+		viper.Set("focus", viper.Get("conformance-image"))
```

Comment:
I think `"\\[Conformance\\]"` is not same as viper.Get("conformance-image"))

Reply:
I must've overlooked this, thanks for pointing out.